src/cmake_modules/GncFindPkgConfig.cmake

```diff
@@ -247,12 +247,10 @@ endmacro()
 
 else()
 
+include(FindPkgConfig)
+
 macro(gnc_pkg_check_modules _prefix _module0)
-  if (NOT DEFINED __pkg_config_checked_${_prefix} OR __pkg_config_checked_${_prefix} LESS ${PKG_CONFIG_VERSION} OR NOT ${_prefix}_FOUND)
```

Comment:
This code was (before this commit) executed for the else() case of CMake version below 3.1. It is the copy/pasted contents of the PKG_CHECK_MODULES macro from the default FindPkgConfig.cmake file. When doing the change to support 3.5+, I realized I should just call the default PKG_CHECK_MODULES macro directly.